index.yaml

```diff
@@ -952,9 +952,54 @@ entries:
       catalog.cattle.io/upstream-version: 1.3.1
     apiVersion: v1
     appVersion: v1.3.1
-    created: "2022-08-11T21:46:43.199870903+08:00"
+    created: "2022-08-30T19:22:26.194791063-07:00"
     description: Longhorn is a distributed block storage system for Kubernetes.
-    digest: 38423d679dfdd9a47188a3239f01fdb13053aaa439a6f4a3c23af8611254a95a
+    digest: 9677c58e4b34a1488e19a9769925dc15ac327e2bbbde3684f640b0de41fb98f7
+    home: https://github.com/longhorn/longhorn
+    icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.png
+    keywords:
+    - longhorn
+    - storage
+    - distributed
+    - block
+    - device
+    - iscsi
+    - nfs
+    kubeVersion: '>=1.18.0-0'
+    maintainers:
+    - email: maintainers@longhorn.io
+      name: Longhorn maintainers
+    name: longhorn
+    sources:
+    - https://github.com/longhorn/longhorn
+    - https://github.com/longhorn/longhorn-engine
+    - https://github.com/longhorn/longhorn-instance-manager
+    - https://github.com/longhorn/longhorn-share-manager
+    - https://github.com/longhorn/longhorn-manager
+    - https://github.com/longhorn/longhorn-ui
+    - https://github.com/longhorn/longhorn-tests
+    - https://github.com/longhorn/backing-image-manager
+    urls:
+    - assets/longhorn/longhorn-100.2.2+up1.3.1.tgz
+    version: 100.2.2+up1.3.1
+  - annotations:
+      catalog.cattle.io/auto-install: longhorn-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Longhorn
+      catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
+      catalog.cattle.io/namespace: longhorn-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+      catalog.cattle.io/rancher-version: < 2.6.0-0
```

Comment:
Should we also hide longhorn-crd 100.2.2+up1.3.1 as well?

Reply:
`longhorn-crd 100.2.2+up1.3.1` is the fixed version. Do you mean `longhorn-crd 100.2.1+up1.3.1` ? I think we don't need to because the main chart`100.2.1+up1.3.1` is hidden and the user has no way to install the depend longhorn-crd chart